### PR TITLE
Updated for DataGrid in 10 mins

### PR DESCRIPTION
**Pull request by edeandrea**
Some text updates as well as some re-wording and removal of things that might just not belong.

### CODE REVIEW

File: docs/dgin10mins.adoc [hunk 1]
```diff
@@ -7,89 +7,74 @@ image::dg8.png[Red Hat Data Grid 8.0, 700]
...
+2. Resiliencey in case of failures
+3. Processing efficency and performance
+4. Events, streams, and distribution capabilites
 
 
 image::caching.png[Caching, 700]
 
 
-By having such capabilites a cache is no longer just a datastructure in the memory, but also as a developer now you have the possiblity to take this component out of your local in memory processing and distribute it out on the network. Thereby incase of application failures you will still be able to access this data from the last point where you left off. 
+By having such capabilites a cache is no longer just a data structure in memory, but also as a developer now you have the possiblity to take this component out of your local in memory processing and distribute it out on the network. Thereby in case of application failures you will still be able to access this data from the last point where you left off.
```
Comment (sshaaf) [inline docs/dgin10mins.adoc]: suggestion: cache is no longer just an in memory data structure, ..

**Review by sshaaf — COMMENTED** [review]
Thanks! also made one suggestion to this.

File: docs/dgin10mins.adoc (hunk 1 above)
Comment (edeandrea) [reply docs/dgin10mins.adoc]: Pushed a new commit based on your suggestion.

**Comment by edeandrea** [comment]
@sshaaf I rebased into a single commit

**Comment by sshaaf** [comment]
Thanks @edeandrea

**Pull request merged**